src/adapter/graphql/tests/tests/test_gql_dataset_flow_configs.rs

```diff
@@ -1572,19 +1576,24 @@ impl FlowConfigHarness {
     }
 
     async fn create_root_dataset(&self) -> CreateDatasetResult {
+        let publicly_available = true;
+
         self.dataset_repo
             .create_dataset_from_snapshot(
                 MetadataFactory::dataset_snapshot()
                     .kind(DatasetKind::Root)
                     .name("foo")
                     .push_event(MetadataFactory::set_polling_source().build())
                     .build(),
+                publicly_available,
             )
             .await
             .unwrap()
     }
 
     async fn create_derived_dataset(&self) -> CreateDatasetResult {
+        let publicly_available = true;
```

Comment:
Should we have a `Visibility::Public` enum to avoid the problem of unnamed bool literals?

Reply:
`Default::default()` is now used in all tests